OptimizelySDK.Tests/ConfigTest/HttpProjectConfigManagerTest.cs

```diff
@@ -129,88 +157,111 @@ public void TestOnReadyPromiseResolvedImmediatelyWhenDatafileIsProvided()
                 .WithStartByDefault()
                 .Build();
 
-            stopwatch.Start();
             httpManager.OnReady().Wait();
-            stopwatch.Stop();
 
-            Assert.True(stopwatch.Elapsed.Seconds == 0);
-            Assert.NotNull(httpManager.GetConfig());
+            Assert.AreEqual("15", httpManager.GetConfig().Revision);
+
+            // loaded datafile from config manager after a second.
+            t.Wait();
+            Task.Delay(200).Wait();
```

Comment:
Heh i am not comfortable with us introducing all these arbitrary waits. Is there something else we can use as a trigger? Like maybe wait for a config-update notif?